modules/instrument_manager/php/NDB_Menu_Filter_instrument_manager.class.inc

```diff
@@ -97,8 +97,10 @@ class NDB_Menu_Filter_Instrument_Manager extends NDB_Menu_Filter
                 );
 
                 exec(
```

Comment:
Is there a reason that this class uses manual queries at all instead of the `Database` class? Seems like a weird choice.

This might be a good opportunity to refactor while we're updating this code (unless somebody knows a reason why it's written this way..)